### PR TITLE
General polish

### PR DESCRIPTION
**Pull request by trevordmiller**
# Changes

- Update instructor revenue card to latest mocks
- Update instructor course stats to latest mocks

# Result For User

The stats show what is available; wont display at all if instructor is unpublished, only displays courses if they exist etc.

lesson-stats
--------
lesson-course-stats-generic
--------
lesson-course-stats
--------
lesson-course-stats-2

mobile-minutes
--------
mobile-revenue
--------
desktop-minutes
--------
desktop-revenue

---

### CODE REVIEW

File: src/components/InstructorLessonsCard/index.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ import TitleCard from 'components/TitleCard'
 import LessonGroups from 'components/LessonGroups'
 
 export default ({instructor}) => (
-  <Maybe condition={instructor}>
+  <Maybe condition={Boolean(instructor)}>
```
Comment (trevordmiller) [inline src/components/InstructorLessonsCard/index.js:8]: Maybe requires a boolean, not just a truthy value

File: src/components/InstructorRevenueCard/components/LineChart/index.js [hunk 2]
```diff
@@ -4,41 +4,40 @@ import hexToRgba from 'hex-rgba'
 import {Line} from 'react-chartjs-2'
 import numberFormattingByType from 'utils/numberFormattingByType'
 
-const sharedOptions = {
+const tooltipColor = '#63768d' // # dark-gray
```
Comment (trevordmiller) [inline src/components/InstructorRevenueCard/components/LineChart/index.js:7]: Will be able to replace hard-coded color here once https://github.com/eggheadio/tachyons-egghead/issues/28 is done

**Comment by vojtaholik** [comment]
This is looking sweet!

File: src/components/InstructorRevenueCard/index.js [hunk 3]
```diff
@@ -1,15 +1,20 @@
...
 import OpenToggle from 'components/OpenToggle'
 import currentMonthStartDate from './utils/currentMonthStartDate'
 import totalRevenue from './utils/totalRevenue'
 import removeRevenueMonth from './utils/removeRevenueMonth'
+import prettyMonthName from './utils/prettyMonthName'
 import RevenuePeriod from './components/RevenuePeriod'
 import LineChart from './components/LineChart'
 
+const revenueColor = '#59cd90' // # blue
+const minutesColor = '#B0B6BE' // # dark-gray-secondary
```
Comment (trevordmiller) [inline src/components/InstructorRevenueCard/index.js:15]: Will be able to replace these hard coded colors here once https://github.com/eggheadio/tachyons-egghead/issues/28 is done

File: src/components/InstructorRevenueCard/index.js [hunk 4]
```diff
@@ -23,95 +28,81 @@ export default ({revenueUrl}) => (
           return null
         }
 
-        const currentMonthDates = map(data, month => month.month)
+        const currentMonthNames = map(data, month => prettyMonthName(month.month))
```
Comment (trevordmiller) [inline src/components/InstructorRevenueCard/index.js:31]: Made new `prettyMonthName` module that converts a `yyyy-mm-dd` string to `Jan`, `Feb` etc.:

image

File: src/components/InstructorRevenueCard/index.js [hunk 5]
```diff
@@ -23,95 +28,81 @@ export default ({revenueUrl}) => (
...
-                            values={[currentTotalRevenue.monthCount]} 
-                          />}
-                          revenue={currentTotalRevenue.revenue}
-                          subscriberMinutes={currentTotalRevenue.minutes_watched}
-                        />
+                  <div 
+                    className='pa5 nowrap-l'
+                    style={{
+                      boxShadow: '10px 0px 10px -10px rgba(0, 0, 0, 0.1)',
+                    }}
```
Comment (trevordmiller) [inline src/components/InstructorRevenueCard/index.js:45]: The right-side only shadow from the mocks wasn't possible with tachyons; we may want to promote this to `tachyons-egghead` if it is going to be reused

Comment (vojtaholik) [inline src/components/InstructorRevenueCard/index.js]: could you try this pretty please?  
**boxShadow: '10px 0px 25px -10px rgba(35, 45, 59, 0.1)'**
rgba==$base

**Review by tayiorbeii — APPROVED** [review]

File: src/components/InstructorRevenueCard/index.js (hunk 5 above)
Comment (vojtaholik) [reply src/components/InstructorRevenueCard/index.js:45]: @trevordmiller Could you try this pretty please?

boxShadow: '10px 0px **25px** -10px rgba(35,45,59, 0.1)'

rgba==$base color

Comment (trevordmiller) [reply src/components/InstructorRevenueCard/index.js:45]: @vojtaholik Yeah no problem. This PR is already merged so I made a new story for some quick style tweaks that I'll get to later if you want to add to it; basically any small text/color/padding type things that can be done with CSS values are fair game for this story:
https://github.com/eggheadio/egghead-instructor-center/issues/216

**Comment by vojtaholik** [comment]
Would it be possible to center all text (switch and stats) on mobile view? :) 
image


I think that'd be all from me, plus that box shadow comment from yesterday.
Looking great! 👍

**Comment by trevordmiller** [comment]
@vojtaholik Yeah no problem, added to the quick style tweaks story: 
https://github.com/eggheadio/egghead-instructor-center/issues/216

**Pull request merged**